de_project/de_app/data_import/unr-covid-scraper.py

```diff
@@ -0,0 +1,35 @@
+from bs4 import BeautifulSoup as BS
+from ..models import RecordHandler, Record, Metric
+
+
+####                   THIS IS EXAMPLE ONLY
+##
+## UNR changed their webpage, so this won't work
+##
+####
+
+class UNRCovidScraper:
```

Comment:
How would this code be executed? It doesn't look like anyone instantiates this class

Reply:
It's not!

Reply:
So how to you intend to run any actual scrapers?